airflow/api_fastapi/execution_api/routes/task_instances.py

```diff
@@ -48,6 +51,108 @@
 log = logging.getLogger(__name__)
 
 
+@router.patch(
+    "/{task_instance_id}/run",
```

Comment:
Maybe 
```suggestion
    "/{task_instance_id}/start",
```

Reply:
Thinking more about this, maybe run is better, as this could be called more than once in the case of things like reschedule couldn't it? (Yes, it's just naming so it doesn't matter, but names also do matter)

Reply:
Yeah, precisely -- reschedule/defer